app/views.py

```diff
@@ -3,8 +3,26 @@
 from app.models import Users
 from app import app
 
+# Init
+sample_account = Users()
+
 
 @app.route('/api/v1/register', methods=['POST'])
 def register():
```

Comment:
the function doesn't check whether the user has inputted any data in their payload before they make the post, also doesnt check whether the user already exists before they can be appended to the dictionary

Reply:
Will do